pkl/version.go

```diff
@@ -0,0 +1,107 @@
+package pkl
+
+import (
+	"fmt"
+	"strconv"
+)
+
+//goland:noinspection GoSnakeCaseUsage
+var pklVersion0_25 = mustParseSemver("0.25.0")
+
+//goland:noinspection GoSnakeCaseUsage
+var pklVersion0_26 = mustParseSemver("0.26.0")
+
+type semver struct {
+	major      int
+	minor      int
+	patch      int
+	prerelease string
+	build      string
+}
+
+func compareInt(a, b int) int {
+	if a == b {
+		return 0
+	}
+	if a < b {
+		return -1
+	}
+	return 1
+}
+
+func (s *semver) compareToString(other string) int {
+	otherVersion, err := parseSemver(other)
+	if err != nil {
+		return 0
```

Comment:
Returning `0` (equals) for bogus versions seems prone to bugs. Would be better to return `int, err` and let the caller check for errors if they want. You could still ignore the error if you don't care.